scripts/codegen-operator

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source $(dirname $0)/lib/debug_functions
+
+GOPATH="${GOPATH:-~/go}"
+GOFLAGS=""
+
+apt-get update -y || true
+apt-get install -y make mercurial cvs || dnf install make mercurial cvs
+mkdir -p $GOPATH/bin
+mkdir -p /root/go/bin
+
+export PATH=$PATH:$GOPATH/bin:/root/go/bin
+
+OP_FRAMEWORK="$GOPATH/src/github.com/operator-framework"
+
+mkdir -p $OP_FRAMEWORK
+pushd $OP_FRAMEWORK
+
+kind delete cluster || true
```

Comment:
Why is a kind cluster needed here? This logic is already in the gen script..

Reply:
To compiler operator-sdk it's also necessary... not to install from binary, but binary does not work.